core/src/main/java/io/questdb/cutlass/pgwire/PGConnectionContext.java

```diff
@@ -2453,6 +2453,20 @@ private void sendAndReset() throws PeerDisconnectedException, PeerIsSlowToReadEx
         responseAsciiSink.reset();
     }
 
+    private void sendAndResetWait() throws PeerDisconnectedException {
+        // This is simplified waited send for very limited use cases where introducing another state is an overkill.
+        // This method busy waits to send buffer.
+        while (true) {
```

Comment:
Would be nice to check the circuit breaker around this place to avoid a potentially infinite loop. WDYT?

Reply:
good idea

Reply:
done